tests/unit/test_models_rates.py

```diff
@@ -134,13 +134,35 @@ def test_rates_staticmethod_get_expected_and_extras(sample_rates_dict):
 def test_rates_method_to_raw(sample_rates_dict, sample_rates_txt):
     rates = RatesStatus.from_dict(sample_rates_dict)
 
-    assert rates.to_raw() == sample_rates_txt
+    assert (
+        rates.to_raw()
+        == """\
+TamingSpeedMultiplier=3.0
+HarvestAmountMultiplier=3.0
+XPMultiplier=3.0
+MatingIntervalMultiplier=0.6
+BabyMatureSpeedMultiplier=3.0
+EggHatchSpeedMultiplier=3.0
+BabyCuddleIntervalMultiplier=0.6
+BabyImprintAmountMultiplier=3.0
+HexagonRewardMultiplier=1.5"""
+    )
```

Comment:
@travipross 

Since this output should no longer match `sample_rates_txt`, yet `sample_rates_text` is still required for other tests, I decided to explicitly define it rather than using a fixture. What are your thoughts on this approach? Would you have added a second fixture with the extra parameter removed?

Reply:
Yeah, this was pretty loosey-goosey usage of fixtures on my part anyway; very few of them were reused so they almost always could have been defined as string literals directly in the corresponding test. Totally a matter of opinion and preference, so if it's testing what you want it to, and it's passing, then it's a good test.